R/gitignore.R

```diff
@@ -72,7 +72,10 @@ gitignore_content_root <- function(root_path) {
 
 
 gitignore_content_src <- function(name, root_path) {
-  dat <- orderly_read_r(file.path(root_path, "src", name, "orderly.R"))
+  src <- file.path(root_path, "src", name)
+  orderly_name <- deprecate_old_orderly_name(src, name)
+  dat <- orderly_read_r(file.path(root_path, "src", name, orderly_name),
+                        orderly_name)
```

Comment:
```suggestion
  dat <- orderly_read(file.path(root_path, "src", name))
```

Unless I'm missing something, there's no much of a reason to use `orderly_read_r` here, neither then nor nor. It's basically the same code as in that function.

Reply:
true yh we could just have `orderly_read` do the deprecation check, done